[India] Added Data Portal India

yasarshaikh opened the pull request:
Provided Repository is not owned/associated by me however, it's associated by Govt. 

URL for DataPortalIndia
GitHub - https://github.com/DataPortalIndia 
Official Site - https://data.gov.in/  
Email - [ndsap@gov.in](mailto:ndsap@gov.in)
  
 :heart: GitHub Government Contributors

yasarshaikh committed:
Added ONDC & Data Portal India for the India

mlinksva commented:
It looks like ONDC is a non-profit (section 8) organization rather than a government organization. I will merge the DataPortalIndia suggestion. Please feel free to correct me!

mlinksva reviewed (commented):
sort, remove ONDC as not a government organization afaict

mlinksva reviewed (changes requested):
sort, remove ONDC as not a government organization afaict

yasarshaikh committed:
Removed ONDC from India list

Removed ONDC from India list as it's not directly associated with govt
Also, kept the list alphabetically

yasarshaikh commented:
@mlinksva - Updated the file as suggested. Please check now and let me know if any other changes are required.

mlinksva commented:
Thanks! Apologies for the noise above. I was trying to use the `.` code editor (first time user in context of a PR) to make changes but did not realize it could only (in this context) add to a review.

mlinksva commented on _data/governments.yml:
```diff
@@ -303,8 +303,10 @@ Hungary:
+  - DataPortalIndia
   - ISRO
   - itschool
+  
```
Could you delete this extra line? Thanks!

yasarshaikh replied:
Deleted, thanks!

yasarshaikh committed:
Removed unnecessary whitespace character/lines
